packages/accounts-password/password_tests.js

```diff
@@ -1697,6 +1697,34 @@ if (Meteor.isServer) (function () {
     }
   )
 
+  Tinytest.add(
+    'passwords - enroll tokens don\'t get cleaned up when reset tokens are cleaned up',
+    function (test) {
+      var email = test.id + '-intercept@example.com';
+      var userId = Accounts.createUser({email: email, password: 'password'});
+      Accounts.sendEnrollmentEmail(userId, email);
+      test.isTrue(!!Meteor.users.findOne(userId).services.password.reset);
```

Comment:
I mean that when line 1710 runs `Meteor.users.findOne(userId).services.password.reset`, it's meant to make sure that it's the same as when it ran on line 1706, right?